packages/models-library/src/models_library/project_networks.py

```diff
@@ -0,0 +1,55 @@
+from models_library.projects import ProjectID
+from pydantic import BaseModel, Field, constr
+
```

Comment:
sorry to be a pain in the *#$, but ALL the other table are called ```projectS``` why this one not? Would it be possible to keep this the same everywhere?

Reply:
In my mind the name makes perfect sense. This table contains "the project's networks", thus the resulting name `project_networks`. Am I doing the plural wrong?

Reply:
no, but the tables were made like this... and as a matter of fact all the other modules in models_library as well. so I propose to either rename them all or keep with the flow.

Reply:
I see, I've missed this. Let's stick with the flow. Going for projects_networks. Also the model will get renamed at this point.